src/tests/FunctionalTests/wasm/AOT/browser/Wasm.Aot.Browser.Test.csproj

```diff
@@ -10,6 +10,16 @@
     <Content Include="index.html">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <WasmExtraFilesToDeploy Include="index.html" />
   </ItemGroup>
+
+  <Target Name="AfterWasmBuildApp" AfterTargets="WasmBuildApp">
+    <Copy SourceFiles="$(OutDir)\index.html" DestinationFolder="$(WasmAppDir)" />
+  </Target>
+
+  <Target Name="SetMainAssemblyPath" BeforeTargets="Publish">
+    <PropertyGroup>
+      <WasmMainAssemblyPath>$(PublishDir)Wasm.Interpreter.Browser.Test.dll</WasmMainAssemblyPath>
```

Comment:
Change this to `Wasm.Aot.Browser.Test.dll`.

Reply:
yeah, copy-paste, thanks